### DISCUSSION TITLE
pscript中的login方法可否提供一些案例

### DISCUSSION THREAD

**Discussion by claywong** [ctripcorp/flybirds#148] [Q&A] [answered]
我的想法是login自己调用登陆接口，获取cookie，写入到set_global_cache
可惜Init browse在login调用之前，没办法在初始化的时候配置login的cookie

**Comment by clgwlg** [comment, marked as answer]
看看是否可以参考这个例子
https://github.com/ctripcorp/flybirds/releases/tag/v0.4.8
before_all_extend的触发时机再init browser之前

**Reply by claywong** [reply]
我在login，set_global_cache cookie之后，re_init_page()，解决了这个问题

```
def login(user_name, password):
    GlobalContext.set_global_cache("cookies", [
        {"name": "PHPSESSID", "value": "xxxxxxxxxx", "path": "/", "domain": "example.com"}])
    re_init_page()
    pass
```
